_sass/_variables.scss

```
@@ -9,7 +9,7 @@ $spacing-unit: $base-line-height * 1rem;
 
 $text-color: #111;
 $background-color: #ffffff;
-$background-color-header: #313131;
+$background-color-header: #029cd3;
```

Tou optimalizací se malinko změnila barva 
```suggestion
$background-color-header: #179BD3;
```